app/directives/jqTree.directive.js

```diff
@@ -1,27 +1,14 @@
 'use strict';
 
 (function() {
-  angular.module('treeView', []).directive('tree', function($parse, SelectedFiles) {
-    var iterate_and_apply_event_recursively = function(event, node, tree) {
```

Comment:
Why move this? It seems like a useful piece of generic functionality to be able to get all nodes modified by an event recursively applied.